apps/rpc/src/index.ts

```diff
@@ -98,7 +100,6 @@ server.get("/health", (_, res) => {
   res.send({ status: "ok" })
 })
 
-verifyAwsCredentials()
-
+await verifyIdentityCredenetials()
```

Comment:
Might as well await here. I don't think it really matters because it's a long-lived service